src/sentry/tasks/integrations.py

```diff
@@ -403,3 +405,41 @@ def vsts_subscription_check(integration_id, organization_id, **kwargs):
 
         integration.metadata["subscription"]["check"] = time()
         integration.save()
+
+
+@instrumented_task(
+    name="sentry.tasks.integrations.sync_status_inbound",
+    queue="integrations",
+    default_retry_delay=60 * 5,
+    max_retries=5,
+)
+@retry(exclude=(Integration.DoesNotExist,))
+@track_group_async_operation
+def sync_status_inbound(
+    integration_id: int, organization_id: int, issue_key: str, data: Mapping[str, Any]
```

Comment:
I agree that providing a formless blob of data isn't ideal here, but since the task doesn't directly handle or 'expect a shape' from the data, I think it should be fine.